docs/features/classes.md

```diff
@@ -0,0 +1,16 @@
+# Classes
+
+```js
+// This will create a simple public class that takes in two ints and has a public function and a private function
+class pub Test [a:int b:int] {
+	let pub sum = [] -> int {
+		return a + b
+	} // This uses a and b from the first line of the class as they are internal variables
+
+	let internalSum = a + b // This is private
+}
+```
+
+## Notes:
+- There is currently no way for multiple contructors.
```

Comment:
Could suggest using static methods by means of functions instead, and give rationale for not having multiple constructors (I believe it's because Python doesn't have any)

Reply:
I think that is not the best for a simple description as it might be better for them to choose their own way to do it because you could also use currying or `maybe` types to deal with that.